addons/l10n_account_edi_ubl_cii_tests/tests/common.py

```diff
@@ -234,8 +234,13 @@ def _assert_invoice_attachment(self, attachment, xpaths, expected_file):
         xml_content = base64.b64decode(attachment.with_context(bin_size=False).datas)
         xml_etree = self.get_xml_tree_from_string(xml_content)
 
-        expected_file_path = get_resource_path('l10n_account_edi_ubl_cii_tests', 'tests/test_files', expected_file)
-        expected_etree = etree.parse(expected_file_path).getroot()
+        # Extract the module name from the following patterns:
+        # odoo.addons.l10n_account_edi_ubl_cii_tests.tests.test_xml_cii_fr
+        # odoo.addons.l10n_dk_oioubl.tests.test_xml_oioubl_dk
+        module = self.__module__.split('.')[2]
```

Comment:
I think you can use `self.test_module` coming from [Metacase class](https://github.com/odoo/odoo/blob/6d5d234f15e458c4a92c240fc743aee941b460e2/odoo/tests/common.py#L222)

Reply:
Thanks!